recipe/Makefile.vc

```diff
@@ -0,0 +1,257 @@
+# To build locally get a copy of the dependancies (http://windows.php.net/downloads/php-sdk/deps/)
+# These are the dependencies used by PHP and largely used and tested. These binaries are built using
+# the sources available at https://github.com/winlibs
+# Then run 'nmake /f windows\Makefile.vc' in MS dev console
+# To build tests as well 'nmake /f windows\Makefile.vc all'
+# To build tests and execute them 'nmake /f windows\Makefile.vc check'
+
+CC=cl
+LD=link
+LB=lib
+
+!IFNDEF DEBUG
+DEBUG=0
+!ENDIF
+
+!IFNDEF WITH_DEVEL
+WITH_DEVEL=I$(CONDA_PREFIX)\Library\
+!ENDIF
+
+!IFNDEF WITH_BUILD
+WITH_BUILD=.\gdbuild\
+!ENDIF
+
+LIBGD_SRC_DIR=src
+LIBGD_OBJ_DIR=$(WITH_BUILD)
+LIBGD_BIN_DIR=$(WITH_BUILD)\bin
+LIBGD_TEST_DIR=test
+
+!IFNDEF PLATFORM
+PLATFORM=x86
+!ENDIF
+# detect x64 cross-compiler env
+!IF "$(PREFERREDTOOLARCHITECTURE)" == "x64"
+PLATFORM=x64
+!ENDIF
+PLATFORM=$(PLATFORM:X=x)
+
+BUILD_DIR=$(WITH_BUILD)
+
+!MESSAGE library source:  $(LIBGD_SRC_DIR)
+
+GD_DLL=libgd.dll
+GD_LIB=libgd_a.lib
+GD_CONFIG_H=$(LIBGD_SRC_DIR)\config.h
+
+
+SDR=$(BUILD_DIR)^\
+TDR=$(BUILD_DIR)\tests^\
+TARGETDLL=$(BUILD_DIR)\$(GD_DLL)
+TARGETLIB=$(BUILD_DIR)\$(GD_LIB)
+
+LIB_OBJS= \
+  $(LIBGD_OBJ_DIR)\gd.obj \
+  $(LIBGD_OBJ_DIR)\gdcache.obj \
+  $(LIBGD_OBJ_DIR)\gd_color.obj \
+  $(LIBGD_OBJ_DIR)\gd_color_match.obj \
+  $(LIBGD_OBJ_DIR)\gdfontg.obj \
+  $(LIBGD_OBJ_DIR)\gdfontl.obj \
+  $(LIBGD_OBJ_DIR)\gdfontmb.obj \
+  $(LIBGD_OBJ_DIR)\gdfonts.obj \
+  $(LIBGD_OBJ_DIR)\gdfontt.obj \
+  $(LIBGD_OBJ_DIR)\gdft.obj \
+  $(LIBGD_OBJ_DIR)\gdfx.obj \
+  $(LIBGD_OBJ_DIR)\gd_filename.obj \
+  $(LIBGD_OBJ_DIR)\gd_filter.obj \
+  $(LIBGD_OBJ_DIR)\gd_bmp.obj \
+  $(LIBGD_OBJ_DIR)\gd_gd2.obj \
+  $(LIBGD_OBJ_DIR)\gd_gd.obj \
+  $(LIBGD_OBJ_DIR)\gd_gif_in.obj \
+  $(LIBGD_OBJ_DIR)\gd_gif_out.obj \
+  $(LIBGD_OBJ_DIR)\gdhelpers.obj \
+  $(LIBGD_OBJ_DIR)\gd_io.obj \
+  $(LIBGD_OBJ_DIR)\gd_io_dp.obj \
+  $(LIBGD_OBJ_DIR)\gd_io_file.obj \
+  $(LIBGD_OBJ_DIR)\gd_io_ss.obj \
+  $(LIBGD_OBJ_DIR)\gd_jpeg.obj \
+  $(LIBGD_OBJ_DIR)\gd_security.obj \
+  $(LIBGD_OBJ_DIR)\gd_tga.obj \
+  $(LIBGD_OBJ_DIR)\gd_tiff.obj \
+  $(LIBGD_OBJ_DIR)\gd_xbm.obj \
+  $(LIBGD_OBJ_DIR)\gdkanji.obj \
+  $(LIBGD_OBJ_DIR)\gd_nnquant.obj \
+  $(LIBGD_OBJ_DIR)\gd_png.obj \
+  $(LIBGD_OBJ_DIR)\gd_ss.obj \
+  $(LIBGD_OBJ_DIR)\gdtables.obj \
+  $(LIBGD_OBJ_DIR)\gd_topal.obj \
+  $(LIBGD_OBJ_DIR)\gd_transform.obj \
+  $(LIBGD_OBJ_DIR)\gd_wbmp.obj \
+  $(LIBGD_OBJ_DIR)\gdxpm.obj \
+  $(LIBGD_OBJ_DIR)\wbmp.obj \
+  $(LIBGD_OBJ_DIR)\gd_interpolation.obj \
+  $(LIBGD_OBJ_DIR)\gd_matrix.obj \
+  $(LIBGD_OBJ_DIR)\gd_rotate.obj \
+  $(LIBGD_OBJ_DIR)\gd_version.obj \
+  $(LIBGD_OBJ_DIR)\gd_crop.obj \
+  $(LIBGD_OBJ_DIR)\gd_color_map.obj \
+  $(LIBGD_OBJ_DIR)\gd_webp.obj
+
+LIBS=kernel32.lib ole32.lib user32.lib advapi32.lib shell32.lib ws2_32.lib Dnsapi.lib Gdi32.Lib
+LIBS_GD=libjpeg.lib freetype.lib libpng.lib iconv.lib zlib.lib libwebp.lib libxpm.dll.a tiff.lib
```

Comment:
have you tried linking against the dlls instead? itwould be a bit more "standard" for conda-forge to have stuff dynamically linked.

Reply:
I have not tried that yet. I'll reconfigure the Makefile.vc to see if that works.